core/domain/collection_domain.py

```diff
@@ -376,15 +376,30 @@ def __init__(self, skill_id, name, question_ids):
         self.question_ids = question_ids
 
     def to_dict(self):
-        # Note to developers: ensure this matches the frontend in
-        # CollectionSkillObjectFactory
+        """Returns the dict of CollectionSkill object.
+
+        Note to developers: Ensure this matches the frontend in
+        CollectionSkillObjectFactory.
+
+        Returns:
+            dict. The dict of CollectionSkill object.
+        """
         return {
             'name': self.name,
             'question_ids': self.question_ids
         }
 
     @classmethod
     def from_dict(cls, skill_id, skill_dict):
+        """Returns the CollectionSkill object.
+
+        Args:
+            skill_id: str. The skill ID.
+            skill_dict: dict. The skill dict.
+
+        Returns:
+            Object. The CollectionSkill object.
```

Comment:
Wrong type.

Reply:
Sorry, it has been corrected to `CollectionSkill`.